src/main/reactjs/src/redux/selectors/clerkTranslator.ts

```diff
@@ -64,13 +89,23 @@ export const selectFilteredSelectedTranslators = createSelector(
 
 // Helpers
 
-const isAuthorisationValid = (authorisation: Authorisation, now: Date) => {
+const expiringSoonTreshold = (currentDate: Date) => {
+  const expiringSoonThreshold = DateUtils.dateAtStartOfDay(currentDate);
+  expiringSoonThreshold.setMonth(expiringSoonThreshold.getMonth() + 3);
+
+  return expiringSoonThreshold;
+};
+
+const isAuthorisationValid = (
+  authorisation: Authorisation,
+  currentDate: Date
+) => {
   const term = authorisation.effectiveTerm;
   if (!term || !term.end) {
     return true;
   }
 
-  return now < term.end;
+  return currentDate <= term.end;
 };
 
 const isAuthorisationExpiringSoon = (
```

Comment:
Tried changing the return value here to
`return DateUtils.dateAtStartOfDay(term.end) <= expiringSoonThreshold;`.

That properly returned (on current date, 26.1.) translators which have authorisations expiring on 25.4. but not those who also have those expiring on 26.4. which would be proper logic. Not a very big deal though.

Reply:
Played around a bit with the dates, and came to the conclusion that one culprit behind these off-by-one errors is that I used the UTC flavored getters of Date. I will change the code to use just local dates for now. This should work better anyway, since the backend always returns just the date parts (without timestamps and timezones) anyways -> makes sense for the frontend as well to deal in purely local dates.